unittest/joint-configurations.cpp

```diff
@@ -186,48 +216,9 @@ struct TestDifferentiationJoint
 
     TV qdot = jmodel.difference(q0,q1);
 
-    BOOST_CHECK_MESSAGE( jmodel.integrate(q0, qdot).isApprox(q1), std::string("Error in difference for joint " + jmodel.shortname()));
+    BOOST_CHECK_MESSAGE( jmodel.isSameConfiguration(jmodel.integrate(q0, qdot), q1), std::string("Error in difference for joint " + jmodel.shortname()));
 
-    BOOST_CHECK_MESSAGE( jmodel.integrate(q1, -qdot).isApprox(q0), std::string("Error in difference for joint " + jmodel.shortname()));
-
-  }
-
-  void operator()(JointModelBase<JointModelFreeFlyer> & jmodel)
```

Comment:
This was a specialization because of quaternions. Now that isSameConfiguration algo is available, this can be removed.